yt_dlp/extractor/thirtydaysinger.py

```diff
@@ -0,0 +1,102 @@
+from .wistia import WistiaIE
+from ..utils import (
+    clean_html,
+    get_elements_html_by_class
+)
+
+
+class ThirtyDaySingerBase(WistiaIE):
```

Comment:
For viz: it inherits from the `WistiaIE` extractor

Reply:
Subclass from `WistiaBaseIE` instead. Don't subclass from concrete IEs

Reply:
Done!